docs/OV_Runtime_UG/supported_plugins/GPU.md

```diff
@@ -258,7 +288,51 @@ The code snippet below demonstrates how to use dynamic batching in simple scenar
          :fragment: dynamic_batch
 
 
-For more details, see the :doc:`dynamic shapes guide<openvino_docs_OV_UG_DynamicShapes>`.
+Notes for performance and memory consumption in dynamic shape
+-----------------------------------------------------------
+
+- Extra CPU utilization during inference :
```

Comment:
Does it make sense to add "Extra GPU utilization" section? During inference dynamic shapes may trigger additional weights reordering, unfused subgraph execution and extra Device <-> Host memory transfers

Reply:
Do you mean "CPU" not "GPU" ? weight reordering, extra device <=> host memory transfer can be added of course, I will do

Reply:
Hmm I think only "extra weight reordering" can be regarded as extra "GPU" utilization but "subgraph execution" is not necessarily regarded as "extra". Also, I think we dont' need to emphasize any "GPU" usage as "extra", we can consider all GPU latencies as a final GPU overhead. The reason to mention CPU usage as "extra" is because CPU is not basically expected to be used.

Reply:
Hmm, agree, thanks!